### ISSUE TITLE
file-header rule option "enforce-trailing-newline" is always violated

### ISSUE THREAD

**Issue by jomaguma** [palantir/tslint#4640]
### Bug Report

- __TSLint version__: 5.15.0
- __TypeScript version__: 3.4.1
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
/* Copyright 2019 */

class Foo {}
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "file-header": [true, "Copyright \\d{4}", "Copyright 2019", "enforce-trailing-newline"]
  }
}
```

#### Actual behavior
```
ERROR: foo.ts:1:1 - missing new line following the file header
```

#### Expected behavior
no error
